AlamofireObjectMapper.podspec

```diff
@@ -1,22 +1,22 @@
 Pod::Spec.new do |s|
 
   s.name = "AlamofireObjectMapper"
-  s.version = "5.2.0"
+  s.version = "5.2.1"
```

Comment:
I think should make this a major version release so we don't force users to use Alamofire 5.0.0

Reply:
Should I change it to 5.3 or 6.0? Sorry for the delay.

Reply:
Lets go with 6.0

Reply:
👍